fedimint-ui/apps/guardian-ui/src/types.tsx

```diff
@@ -47,31 +47,58 @@ export interface Peer {
   status: ServerStatus;
 }
 
+export interface BitcoinRpc {
+  kind: string;
+  url: string;
+}
+
 export type PeerHashMap = Record<string, string>;
 
-export type LnFedimintModule = ['ln', null];
-export type MintFedimintModule = ['mint', { mint_amounts: number[] }];
+export type LnFedimintModule = [
+  'ln',
+  {
+    consensus?: object;
```

Comment:
Do we have plans to make these more concrete types? I imagine this could be done when the modules are no longer hard coded, etc.